tools/gce/linux_performance_worker_init.sh

```diff
@@ -135,6 +135,12 @@ curl -sSL https://get.rvm.io | bash -s stable --ruby
 source ~/.rvm/scripts/rvm
 gem install bundler
 
+# PHP dependencies
```

Comment:
Is what already installed? I copied this line from the Dockerfile but it's possible that some of the apt-get items are already there.